test/dotcom_web/controllers/schedule/trip_info_test.exs

```diff
@@ -578,7 +589,8 @@ defmodule DotcomWeb.ScheduleController.TripInfoTest do
 
   describe "test that wollaston station is properly inserted when expected" do
     test "Does not add Wollaston to non Red line routes", %{conn: conn} do
-      init = init(trip_fn: &trip_fn/2, vehicle_fn: &vehicle_fn/1, prediction_fn: &prediction_fn/1)
+      expect(Predictions.Repo.Mock, :all, &prediction_fn/1)
```

Comment:
Why are we still referencing `prediction_fn`? Should be able to remove that.

Reply:
`prediction_fn/1` is a helper defined in this file. It seemed fine to keep it used here instead of redefining it for every test

Reply:
We shouldn't need it. It looks like it is being used as a kind of factory.